### ISSUE TITLE
Clean up main.py?

### ISSUE THREAD

**Issue by Orinax** [Chong-source/Calendar_Automation_Project#11]
The `main.py` file is not looking as good as it could. Perhaps not all of this should be in the main() function:
```
def main():
    run_program = True
    creds = handle_authentication()

    # Create the names for each individual block that is in the schedule.
    period_block_names = cbb.create_period_block_names(cd.letters_for_period_names,
                                                       cd.periods_per_rotation,
                                                       cd.classes_to_schedule)

    # Assign each rotation day a set of blocks equal to the number of periods per day.
    blocks_of_the_day = cbb.assign_block_names_to_periods(cd.periods_per_day,
                                                          cd.rotation_day_names,
                                                          period_block_names)

    # Create the timetable that holds start and end information for all events in a school day.
    time_schedule = cbb.create_period_times(cd.periods_per_day, cd.start_of_period,
                                            cd.length_of_period, cd.travel_time,
                                            cd.break_after_x_periods, cd.length_of_break,
                                            cd.lunch_after_x_periods, cd.length_of_lunch,
                                            cd.end_of_school_day)

    # Apply rotating day pattern to both semesters, then combine to make a year-long list.
    s1_rotation_days = cbb.apply_rotation_days(cd.semester_1, cd.rotation_day_names)
    s2_rotation_days = cbb.apply_rotation_days(cd.semester_2, cd.rotation_day_names)
    school_year_rotation_days = s1_rotation_days + s2_rotation_days

    # Add a column for RotationDays to the school_calendar_info dataframe.
    cd.df['RotationDays'] = school_year_rotation_days

    # Add a new column for each period. Values in the columns will show which sub blocks are in the period for
    # each specific rotation day.
    for i in range(cd.periods_per_day):
        cd.df[f'Period {i + 1}'] = cd.df['RotationDays']
        for j in range(len(cd.rotation_day_names)):
            cd.df.loc[cd.df['RotationDays'] == cd.rotation_day_names[j],
            f'Period {i + 1}'] = blocks_of_the_day[cd.rotation_day_names[j]][i]

    while run_program:
        welcome_message()
        display_menu(menu_options)
```

Instead, maybe it should be more like this:
```
def main():
    run_program = True
    creds = handle_authentication()

    while run_program:
        welcome_message()
        display_menu(menu_options)
```

If that change is made, that means that all of the lines removed from the main() function will have to be moved to some place else. Moving those lines someplace else could also make it possible to get rid of these import lines, or at least change them:
```
import calendar_building_blocks as cbb
import calendar_details as cd
```

**To summarize:** The `main()` function should be cleaned up / refactored. Maybe some of those lines are placed into functions that can be called? Not sure the best way to handle this at the moment, but it definitely seems like the `main()` function can be improved.

**One last thought:** Maybe there is nothing wrong with the `main()` function the way it is, and maybe there is nothing wrong with those two import lines. At least having another pair of eyes look over this and consider how the function might be changed will be helpful.
